skins/moono-lexicon/mainui.css

```diff
@@ -185,11 +185,11 @@ Special outer level classes used in this file:
 /* Styles for the editable area */
 .cke_editable {
 	background-color: #f1f2f5; /* $light */
-	height: 100%;
+	height: calc(100% - 12px) !important;
 	margin: 0;
 	padding: 12px 16px 8px 16px;
 	position: absolute;
-	width: 100%;
+	width: calc(100% - 12px) !important;
```

Comment:
There are a few [known issues](https://caniuse.com/#search=calc) with `calc()` on IE, but I suspect this usage is fine.

I gather @marcoscv-work already ok'd the usage of `!important` here. You might be able to avoid that by increasing the specificity in another way 🤷 (eg. `.cke_editable.cke_editable` hack), but that obviously won't cut it if you have to beat an ID or an inline style.

Reply:
Yep, actually I wrote this with @marcoscv-work , we need to overwrite inline styling.

Reply:
I totally disagree with the use of `!Important` but in this case, we have not alternative so LGTM